### PR TITLE
Squeeze firmwares to fit the new ja translation

### PR DESCRIPTION
**Pull request by jepler**
#3258 doesn't fit on these boards, so I trimmed them.

The most concerning change (to me) was disabling long int support on the hallowing_m0_express.  If someone can provide a different alternative that passes the TRANSLATION=ja build I'm open to it.

### CODE REVIEW

File: ports/atmel-samd/boards/metro_m0_express/mpconfigboard.mk [hunk 1]
```diff
@@ -12,8 +12,25 @@ EXTERNAL_FLASH_DEVICES = "S25FL216K, GD25Q16C"
 LONGINT_IMPL = MPZ
 
 CIRCUITPY_BITBANGIO = 0
+CIRCUITPY_COUNTIO = 0
+CIRCUITPY_GAMEPAD = 0
```
Comment (dhalbert) [inline ports/atmel-samd/boards/metro_m0_express/mpconfigboard.mk:16]: AUDIOBUSIO is removed from (nearly?) all the other SAMD21 Express builds except for Metro M0. Could we be consistent about what's in generic M0 Express builds? Here instead GAMEPAD was removed, but you left I2CPERIPHERAL.

If we can be consistent, then these defaults could be moved to mpconfigport.h in the SAMD21 conditional section.

File: ports/atmel-samd/boards/circuitplayground_express_displayio/mpconfigboard.mk [hunk 2]
```diff
@@ -12,7 +12,9 @@ EXTERNAL_FLASH_DEVICES = "S25FL216K, GD25Q16C"
 
 # Turn off features and optimizations for Crickit build to make room for additional frozen libs.
 LONGINT_IMPL = NONE
+CIRCUITPY_AUDIOBUSIO = 0
```
Comment (dhalbert) [inline ports/atmel-samd/boards/circuitplayground_express_displayio/mpconfigboard.mk:15]: Is AUDIOBUSIO useful for this CPX build? How does it compare with the other CPX builds, in terms of features?

Comment (jepler) [reply ports/atmel-samd/boards/circuitplayground_express_displayio/mpconfigboard.mk:15]: oh -- yes it is -- PDMIn for the mic :microphone: .  Might have to find something else to cut.  I thought only in terms of I2SOut.

**Comment by jepler** [comment]
The ad-hoc-ness of this does point towards doing something more consistent.  That said, I would like to get the "ja" translation in sooner rather than later...

**Pull request merged**